CHANGELOG.md

```diff
@@ -55,10 +55,14 @@ Users that were registering EIP-165 interfaces with `_registerInterface` from `E
 
 ```solidity
 function supportsInterface(bytes4 interfaceId) public view virtual override returns (bool) {
-    return interfaceId == type(MyInterface).interfaceId || super.supportsInterface(interfaceId);
+  return interfaceId == type(MyInterface).interfaceId || super.supportsInterface(interfaceId);
```

Comment:
Not sure where this change came from it seems unrelated to the merge.

```suggestion
    return interfaceId == type(MyInterface).interfaceId || super.supportsInterface(interfaceId);
```

Reply:
It went from formatting the `CHANGELOG.md` after conflict resolution